addons/delete-others/userscript.js

```diff
@@ -0,0 +1,64 @@
+export default async function ({ addon, console, msg }) {
+  const vm = await addon.tab.traps.vm;
+
+  const types = ["sound", "costume"];
+  let deletedItems = [];
+  let deleted;
+  let target;
+
+  function getRestoreFun(type) {
+    if (type === "costume") {
+      deletedItems.forEach((costume) => {
+        vm.addCostume(costume.md5, costume, target.id, "");
+      });
+    } else if (type === "sound") {
+      deletedItems.forEach((sound) => {
+        vm.addSound(sound, target.id);
+      });
+    }
+    deletedItems = [];
+  }
+
+  addon.tab.createEditorContextMenu(
+    (ctx) => {
+      if (window.confirm(`Are you sure you want to delete all other ${ctx.type}s?`)) {
+        const type = ctx.type === "costume" ? "Costume" : "Sound";
```

Comment:
What's the capitalization for?

Reply:
This should be translatable, with separate strings for costumes and sounds. `addon.tab.confirm` is the preferred way to ask for confirmation.

Reply:
> What's the capitalization for?

Scratch's Redux state for the deletedItem is capitalised 

> This should be translatable, with separate strings for costumes and sounds. `addon.tab.confirm` is the preferred way to ask for confirmation.

Alrighty